source/index.rst

```diff
@@ -89,6 +89,31 @@ These are institutions who were early adopters or provided HPC resources for dev
    applications
    user-documentation
 
+OnDemand System Dependencies
+----------------------------
+
+The following table lists the system dependencies of OnDemand while it is running.  
```

Comment:
I'd replace this line with:

```
For users attempting to install OnDemand from source on non-CentOS/RHEL systems it may be useful to know that OnDemand has several other run and build time dependencies that may need to be installed.
```